### PR TITLE
convert text to single line so tabs in translation issue is fixed

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @MorrisJobke @LukasReschke

### CODE REVIEW

File: settings/templates/personal.php [hunk 1]
```diff
@@ -182,12 +182,7 @@
 	</a>
 
 		<p>
-			<?php print_unescaped($l->t('If you want to support the project
-		<a href="https://nextcloud.com/contribute"
-			target="_blank" rel="noreferrer">join development</a>
-		or
-		<a href="https://nextcloud.com/contribute"
-			target="_blank" rel="noreferrer">spread the word</a>!'));?>
+			<?php print_unescaped($l->t('If you want to support the project <a href="https://nextcloud.com/contribute" target="_blank" rel="noreferrer">join development</a> or <a href="https://nextcloud.com/contribute" target="_blank" rel="noreferrer">spread the word</a>!'));?>
```
Comment (rullzer) [inline settings/templates/personal.php]: Mmm we should split this up. Translating html stuff is bad. I'll take care

File: settings/templates/personal.php [hunk 2]
```diff
@@ -182,12 +182,11 @@
...
-		<a href="https://nextcloud.com/contribute"
-			target="_blank" rel="noreferrer">join development</a>
-		or
-		<a href="https://nextcloud.com/contribute"
-			target="_blank" rel="noreferrer">spread the word</a>!'));?>
+			<?php print_unescaped($l->t('If you want to support the project') .
+				' <a href="https://nextcloud.com/contribute" target="_blank" rel="noreferrer">' .
+				$l->t('join development') . '</a>' . $l->t('or') .
+				'<a href="https://nextcloud.com/contribute" target="_blank" rel="noreferrer">' .
+				$l->t('spread the word') . '</a>' . '!');?>
```
Comment (MorrisJobke) [inline settings/templates/personal.php]: This will not work out with other languages. 😢

**Review by MorrisJobke — CHANGES_REQUESTED** [review]

File: settings/templates/personal.php (hunk 2 above)
Comment (rullzer) [inline settings/templates/personal.php]: Yeah I was still not sure. But adding HTML to the translation string is asking for trouble.

Comment (rullzer) [inline settings/templates/personal.php]: done

**Comment by jancborchardt** [comment]
@rullzer @MorrisJobke does the HTML stuff need to be fixed in the firstrunwizard then as well? https://github.com/nextcloud/firstrunwizard/commit/8a6df421b1a03b265afce744a8f9f81248db8f19

**Comment by rullzer** [comment]
@jancborchardt yeah would be better

**Comment by nickvergessen** [comment]
Looks good 👍

**Comment by jancborchardt** [comment]
@rullzer can you take care of it since you did it here too? :)

**Comment by jancborchardt** [comment]
Also review please @MorrisJobke @rullzer

**Pull request merged**